CHANGELOG.md

```diff
@@ -73,3 +75,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - ITK OpenID Connect: Upgraded from
   `itk-dev/openid-connect` 1.0.0 to 2.1.0
 - OpenId Connect Bundle: Added CLI login feature.
+
+[unreleased]: https://github.com/itk-dev/openid-connect-bundle/compare/3.1.0...HEAD
+[3.1.0]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/3.1.0
+[3.0.3]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/3.0.3
+[3.0.2]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/3.0.2
+[3.0.1]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/3.0.1
+[3.0.0]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/3.0.0
+[2.0.0]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/2.0.0
+[1.1.0]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/1.1.0
+[1.0.1]: https://github.com/itk-dev/openid-connect-bundle/releases/tag/1.0.1
```

Comment:
These should be “[compares](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/viewing-and-comparing-commits/comparing-commits)” between versions (cf. https://keepachangelog.com/), i.e.

```
[3.1.0]: https://github.com/itk-dev/openid-connect-bundle/compare/3.0.3...3.1.0
[3.0.3]: https://github.com/itk-dev/openid-connect-bundle/compare/3.0.2...3.0.3 
…
```

Reply:
Fixed.